code/routers/user/profile.js

```diff
@@ -51,18 +54,12 @@ router.get('/profile', (req, res) => {
           });
         }
         else{
-          userData.push({tripsCount: user[0].count});
+          userData['tripsCount'] = user[0].count;
```

Comment:
Same as the comment above.

Reply:
Same as above. We want to return one JSON object instead of a list of them.